ERD: <text> attribute y: Expected length, "NaN"

**Describe the bug**
When rendering an ERD, the following errors are shown in the console:

```
Error: <text> attribute y: Expected length, "NaN".
Error: <text> attribute y: Expected length, "NaN".
Error: <text> attribute y: Expected length, "NaN".
Error: <text> attribute y: Expected length, "NaN".
Error: <text> attribute y: Expected length, "NaN".
Error: <text> attribute y: Expected length, "NaN".
Error: <text> attribute y: Expected length, "NaN".
Error: <text> attribute y: Expected length, "NaN".
mermaid.min.js:31 Error: <text> attribute y: Expected length, "NaN".
(anonymous) @ mermaid.min.js:31
each @ mermaid.min.js:31
attr @ mermaid.min.js:31
(anonymous) @ mermaid.min.js:41
Ni @ mermaid.min.js:41
render @ mermaid.min.js:41
i @ mermaid.min.js:48
init @ mermaid.min.js:48
```

**To Reproduce**
Steps to reproduce the behavior:
1. Render the following ERD:

```
erDiagram
    ENTITY-1 !-!< ENTITY-2 : role
    ENTITY-1 ?-?< ENTITY-3 : role
```

**Expected behavior**
No errors in console

**Screenshots**
n/a

**Desktop (please complete the following information):**
Irrelevant

**Smartphone (please complete the following information):**
Irrelevant

**Additional context**
Mermaid 8.5.0

--- a/src/diagrams/er/erRenderer.js
+++ b/src/diagrams/er/erRenderer.js
@@ -45,10 +45,13 @@ const drawEntities = function(svgNode, entities, graph) {
       .append('text')
       .attr('id', textId)
       .attr('x', 0)
-      .attr('y', (conf.fontSize + 2 * conf.entityPadding) / 2)
+      .attr('y', 0)
       .attr('dominant-baseline', 'middle')
       .attr('text-anchor', 'middle')
-      .attr('style', 'font-family: ' + getConfig().fontFamily + '; font-size: ' + conf.fontSize)
+      .attr(
+        'style',
+        'font-family: ' + getConfig().fontFamily + '; font-size: ' + conf.fontSize + 'px'
+      )
       .text(id);
 
     // Calculate the width and height of the entity
@@ -227,7 +230,10 @@ const drawRelationshipFromLayout = function(svg, rel, g, insert) {
     .attr('y', labelPoint.y)
     .attr('text-anchor', 'middle')
     .attr('dominant-baseline', 'middle')
-    .attr('style', 'font-family: ' + getConfig().fontFamily + '; font-size: ' + conf.fontSize)
+    .attr(
+      'style',
+      'font-family: ' + getConfig().fontFamily + '; font-size: ' + conf.fontSize + 'px'
+    )
     .text(rel.roleA);
 
   // Figure out how big the opaque 'container' rectangle needs to be

--- a/src/mermaidAPI.js
+++ b/src/mermaidAPI.js
@@ -525,9 +525,9 @@ const config = {
     fill: 'honeydew',
 
     /**
-     * Font size
+     * Font size (expressed as an integer representing a number of  pixels)
      */
-    fontSize: '12px'
+    fontSize: 12
   }
 };
 

